test/functional/neutronless/l2adjacent/Dockerfile

```diff
@@ -22,4 +22,4 @@ RUN pip install --upgrade /root/devenv/f5-openstack-agent
 COPY ./f5-common-python/ /root/devenv/f5-common-python/
 RUN pip install --upgrade /root/devenv/f5-common-python/
 WORKDIR /root/devenv/f5-openstack-agent/test/functional/neutronless/l2adjacent
-RUN py.test -s -v --symbols /root/devenv/f5-openstack-agent/test/functional/neutronless/l2adjacent/conf_symbols.json -v /root/devenv/f5-openstack-agent/test/functional/neutronless/l2adjacent/test_loadbalancer_create.py
+RUN py.test --cov /root/devenv/f5-openstack-agent/f5_openstack_agent -v --symbols /root/devenv/f5-openstack-agent/test/functional/neutronless/l2adjacent/conf_symbols.json -v /root/devenv/f5-openstack-agent/test/functional/neutronless/l2adjacent/test_disconnected_service_creation.py
```

Comment:
Make RUN work on the entire directory and add a comment line with how to run a single suite.